primer3/thermoanalysis.pyx

```diff
@@ -1213,22 +1213,29 @@ cdef class _ThermoAnalysis:
         err_msg = ''
         try:
             global_settings_data = <p3_global_settings*> self.global_settings_data
-            if misprime_lib != None:
-                mp_lib = pdh_create_seq_lib(misprime_lib)
-                if mp_lib == NULL:
-                    err_msg = f'Issue creating misprime_lib {misprime_lib}'
-                    raise ValueError(
-                        f'Issue creating misprime_lib {misprime_lib}'
-                    )
 
-                global_settings_data[0].p_args.repeat_lib = mp_lib
+            if misprime_lib is None:
+                misprime_lib = {}
+
+            mp_lib = pdh_create_seq_lib(misprime_lib)
+            if mp_lib == NULL:
+                err_msg = f'Issue creating misprime_lib {misprime_lib}'
```

Comment:
Looks like this isn't used and can be removed

Reply:
Good point.  I don't believe a `NULL` is ever returned now.

Reply:
updated